.tx/config

```diff
@@ -2,13 +2,13 @@
 host = https://www.transifex.com
 
 [edx-platform.edx-enterprise]
-file_filter = enterprise/conf/locale/<lang>/LC_MESSAGES/django.po
-source_file = enterprise/conf/locale/en/LC_MESSAGES/django.po
+file_filter = enterprise/locale/<lang>/LC_MESSAGES/django.po
```

Comment:
Is it intentional to change the translations file path?

Reply:
Yes, it is intentional. third party apps must have this path for translations otherwise translations do not work.